src/status_im2/contexts/syncing/setup_syncing/view.cljs

```diff
@@ -0,0 +1,133 @@
+(ns status-im2.contexts.syncing.setup-syncing.view
+  (:require [utils.i18n :as i18n]
+            [clojure.string :as string]
+            [quo2.core :as quo]
+            [quo2.foundations.colors :as colors]
+            [react-native.core :as rn]
+            [utils.datetime :as datetime]
+            [status-im2.contexts.syncing.setup-syncing.style :as style]
+            [utils.re-frame :as rf]
+            [status-im2.constants :as constants]
+            [react-native.clipboard :as clipboard]
+            [status-im2.contexts.syncing.sheets.enter-password.view :as enter-password]
+            [status-im2.common.qr-code-viewer.view :as qr-code-viewer]
+            [reagent.core :as reagent]
+            [status-im2.common.resources :as resources]
+            [react-native.hooks :as hooks]))
+
+(def code-valid-for-ms 120000)
+(def one-min-ms 60000)
+
+(defn navigation-bar
+  []
+  [rn/view {:style style/navigation-bar}
+   [quo/page-nav
+    {:align-mid?            true
+     :mid-section           {:type :text-only :main-text ""}
```

Comment:
do we have to pass mid `mid-section` if its an empy string.

Reply:
it seems we don't need mid-section at all 👍